TEMP-CHANGELOG-ALLAUTH.md

```
@@ -0,0 +1,20 @@
+## ???
+
+### ??? Changes
+
+- Added `all-auth` functions: `useAllAuthConfig`, `useAllAuthLogin`, `useAllAuthLoginEmailCode`, `useAllAuthLoginEmailCodeConfirm`, `useAllAuthLoginWebAuthn`, `useAllAuthLogout`, `useAllAuthReauthenticate`, `useAllAuthRecoveryCodeAuthenticate`, `useAllAuthSessions`, `useAllAuthSignUp`, `useAllAuthSignUpWebAuthn`, `useAllAuthTOTPAuthenticate`, `useAllAuthTOTPAuthenticator`, `useAllAuthTOTPAuthenticatorActivate`, `useAllAuthTOTPAuthenticatorDeactivate`, `useAllAuthTOTPReauthenticate`, and `useAllAuthVerifyEmailCodeConfirm`.
+- Added `all-auth` types, constants and services.
+- Removed the `mfa` module.
+- Removed some access functions: `useLogin`, `useLogout`, and `useSignUp`.
+
+----
+
+# @baseapp-frontend/authentication
```

Markdownlint (0.37.0)

Multiple headings with the same content

(MD024, no-duplicate-heading)